tests/functional/functional_tests.py

```diff
@@ -97,55 +93,50 @@ def striptext_in_file(line, file):
         selenium_logger.setLevel(logging.WARNING)
         #chrome_options = webdriver.ChromeOptions()
         #chrome_options.add_experimental_option("mobileEmulation", { "deviceName": "iPhone 7" })
-        self.caps = webdriver.ChromeOptions().to_capabilities()
```

Comment:
Also updated to selenium 4 whilst we're here, so lots of 'find_element_by...' stuff has changed to the new interface